packaging/language/cpanm.py

```diff
@@ -91,29 +91,43 @@
 
 EXAMPLES = '''
 # install Dancer perl package
-- cpanm: name=Dancer
+- cpanm:
+    name: Dancer
 
 # install version 0.99_05 of the Plack perl package
-- cpanm: name=MIYAGAWA/Plack-0.99_05.tar.gz
+- cpanm:
+    name: MIYAGAWA/Plack-0.99_05.tar.gz
 
 # install Dancer into the specified locallib
-- cpanm: name=Dancer locallib=/srv/webapps/my_app/extlib
+- cpanm:
+    name: Dancer
+    locallib: /srv/webapps/my_app/extlib
 
 # install perl dependencies from local directory
-- cpanm: from_path=/srv/webapps/my_app/src/
+- cpanm:
+    from_path: /srv/webapps/my_app/src/
 
 # install Dancer perl package without running the unit tests in indicated locallib
-- cpanm: name=Dancer notest=True locallib=/srv/webapps/my_app/extlib
+- cpanm:
+    name: Dancer
+    notest: True
+    locallib: /srv/webapps/my_app/extlib
 
 # install Dancer perl package from a specific mirror
-- cpanm: name=Dancer mirror=http://cpan.cpantesters.org/
+- cpanm:
+    name: Dancer
+    mirror: 'http://cpan.cpantesters.org/'
 
 # install Dancer perl package into the system root path
-- cpanm: name=Dancer system_lib=yes
+- cpanm:
+    name: Dancer
+    system_lib: yes
 
 # install Dancer if it's not already installed
 # OR the installed version is older than version 1.0
-- cpanm: name=Dancer version=1.0
+- cpanm:
+    name: Dancer
+    version: '1.0'
```

Comment:
It should have been, because as @abadger pointed out that could become a float. I think I was careful on those 'risky' values, but some could have gone through. Do you remember specific cases?